### ISSUE TITLE
1.12 - v1.1.2 Crash on load

### ISSUE THREAD

**Issue by ghost** [Furgl/Stupid-Things#5]
Unable to load into minecraft with the latest version (1.1.2) of the mod, crash report as follows;

```
---- Minecraft Crash Report ----

WARNING: coremods are present:
  PLCore (PassableLeaves-1.12-6.1.0.0.jar)
  LoadingPlugin (Quark-r1.2-104.jar)
  AstralCore (astralsorcery-1.12-1.6.1.jar)
  TNTUtilities Core (tnt_utilities-mc1.12-1.2.2.jar)
  MalisisCorePlugin (malisiscore-1.12-6.0.2.jar)
  MC64836CoreMod (mc64836-0.1.1.jar)
  DynamicSurroundingsCore (DynamicSurroundings-1.12-3.4.6.0.jar)
  DummyPlugin (ForgeEndertech-1.12-3.0.0.0-build.0002.jar)
  LoadingPlugin (NoNausea-MC1.12-1.0.jar)
Contact their authors BEFORE contacting forge

// Don't be sad. I'll do better next time, I promise!

Time: 8/11/17 6:45 PM
Description: There was a severe problem during mod loading that has caused the game to fail

net.minecraftforge.fml.common.LoaderExceptionModCrash: Caught exception from Stupid Things (stupidthings)
Caused by: java.lang.NoClassDefFoundError: scala/actors/threadpool/Arrays
	at furgl.stupidThings.client.ClientProxy.<clinit>(ClientProxy.java:60)
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Unknown Source)
	at net.minecraftforge.fml.common.ProxyInjector.inject(ProxyInjector.java:85)
	at net.minecraftforge.fml.common.FMLModContainer.constructMod(FMLModContainer.java:591)
	at sun.reflect.GeneratedMethodAccessor6.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at com.google.common.eventbus.Subscriber.invokeSubscriberMethod(Subscriber.java:91)
	at com.google.common.eventbus.Subscriber$SynchronizedSubscriber.invokeSubscriberMethod(Subscriber.java:150)
	at com.google.common.eventbus.Subscriber$1.run(Subscriber.java:76)
	at com.google.common.util.concurrent.MoreExecutors$DirectExecutor.execute(MoreExecutors.java:399)
	at com.google.common.eventbus.Subscriber.dispatchEvent(Subscriber.java:71)
	at com.google.common.eventbus.Dispatcher$PerThreadQueuedDispatcher.dispatch(Dispatcher.java:116)
	at com.google.common.eventbus.EventBus.post(EventBus.java:217)
	at net.minecraftforge.fml.common.LoadController.sendEventToModContainer(LoadController.java:252)
	at net.minecraftforge.fml.common.LoadController.propogateStateMessage(LoadController.java:230)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at com.google.common.eventbus.Subscriber.invokeSubscriberMethod(Subscriber.java:91)
	at com.google.common.eventbus.Subscriber$SynchronizedSubscriber.invokeSubscriberMethod(Subscriber.java:150)
	at com.google.common.eventbus.Subscriber$1.run(Subscriber.java:76)
	at com.google.common.util.concurrent.MoreExecutors$DirectExecutor.execute(MoreExecutors.java:399)
	at com.google.common.eventbus.Subscriber.dispatchEvent(Subscriber.java:71)
	at com.google.common.eventbus.Dispatcher$PerThreadQueuedDispatcher.dispatch(Dispatcher.java:116)
	at com.google.common.eventbus.EventBus.post(EventBus.java:217)
	at net.minecraftforge.fml.common.LoadController.distributeStateMessage(LoadController.java:147)
	at net.minecraftforge.fml.common.Loader.loadMods(Loader.java:570)
	at net.minecraftforge.fml.client.FMLClientHandler.beginMinecraftLoading(FMLClientHandler.java:227)
	at net.minecraft.client.Minecraft.func_71384_a(Minecraft.java:466)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:377)
	at net.minecraft.client.main.Main.main(SourceFile:123)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:236)
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:297)
	at org.multimc.EntryPoint.listen(EntryPoint.java:162)
	at org.multimc.EntryPoint.main(EntryPoint.java:53)
Caused by: java.lang.ClassNotFoundException: scala.actors.threadpool.Arrays
	at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:191)
	at java.lang.ClassLoader.loadClass(Unknown Source)
	at java.lang.ClassLoader.loadClass(Unknown Source)
	... 48 more
Caused by: java.lang.NullPointerException


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- System Details --
Details:
	Minecraft Version: 1.12
	Operating System: Windows 8.1 (amd64) version 6.3
	Java Version: 1.8.0_141, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 377077248 bytes (359 MB) / 899284992 bytes (857 MB) up to 4120510464 bytes (3929 MB)
	JVM Flags: 8 total; -XX:SurvivorRatio=2 -XX:+DisableExplicitGC -XX:+UseConcMarkSweepGC -XX:+AggressiveOpts -XX:+UseStringDeduplication -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xms1024m -Xmx4096m
	IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
	FML: MCP 9.40 Powered by Forge 14.21.1.2443 Optifine OptiFine_1.12_HD_U_C4 81 mods loaded, 81 mods active
	States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
	UC	minecraft{1.12} [Minecraft] (minecraft.jar) 
	UC	mcp{9.19} [Minecraft Coder Pack] (minecraft.jar) 
	UC	FML{8.0.99.99} [Forge Mod Loader] (forge-1.12-14.21.1.2443-universal.jar) 
	UC	forge{14.21.1.2443} [Minecraft Forge] (forge-1.12-14.21.1.2443-universal.jar) 
	UC	mc64836coremod{0.1.1} [MC64836 Core Mod] (minecraft.jar) 
	UC	tnt_utilities_core{1.2.2} [TNTUtils Core] (minecraft.jar) 
	UC	additionalbanners{1.5.38} [Additional Banners] (AdditionalBanners-1.12-1.5.38.jar) 
	UC	forgeendertech{1.12-3.0.0.0} [Forge Endertech] (ForgeEndertech-1.12-3.0.0.0-build.0002.jar) 
	UC	adhooks{1.12-3.0.0.0} [Advanced Hook Launchers] (AdHooks-1.12-3.0.0.0-build.0002.jar) 
	UC	aiimprovements{0.0.1.3} [AI Improvements] (AIImprovements-1.12-0.0.1b3.jar) 
	UC	akashictome{1.2-9} [Akashic Tome] (AkashicTome-1.2-9.jar) 
	UC	antiqueatlas{4.4.4} [Antique Atlas] (antiqueatlas-1.12-4.4.4.jar) 
	UC	antiqueatlasoverlay{1.2} [Antique Atlas Overlay] (antiqueatlas-1.12-4.4.4.jar) 
	UC	astralsorcery{1.6.1} [Astral Sorcery] (astralsorcery-1.12-1.6.1.jar) 
	UC	quark{r1.2-104} [Quark] (Quark-r1.2-104.jar) 
	UC	autoreglib{1.3-13} [AutoRegLib] (AutoRegLib-1.3-13.jar) 
	UC	bedbugs{@VERSION@} [Bed Bugs] (BedBugs-1.12-1.0.1.jar) 
	UC	cyclopscore{0.10.11} [Cyclops Core] (CyclopsCore-1.12-0.10.11.jar) 
	UC	capabilityproxy{1.0.0} [CapabilityProxy] (CapabilityProxy-1.12-1.0.0.jar) 
	UC	ceramics{1.12-1.3.2} [Ceramics] (Ceramics-1.12-1.3.2.jar) 
	UC	charcoal_pit{1.4_12} [Charcoal Pit] (CharcoalPit-[1.12]-1.4_12.jar) 
	UC	commoncapabilities{1.3.4} [CommonCapabilities] (CommonCapabilities-1.12-1.3.4.jar) 
	UC	crafttweaker{4.0.2} [CraftTweaker2] (CraftTweaker2-1.12-4.0.2.jar) 
	UC	ctgui{1.0.0} [CT-GUI] (CraftTweaker2-1.12-4.0.2.jar) 
	UC	debugserverinfo{1.0.5} [DebugServerInfo] (DebugServerInfo-1.12-1.0.5.jar) 
	UC	degradeexplodedblocks{1.0.0} [DegradeExplodedBlocks] (degradeexplodedblocks-1.11.2-1.0.0.jar) 
	UC	dirt2path{1.7.0} [Dirt2Path] (dirt2path-1.7.0.jar) 
	UC	doggytalents{1.14.1.240} [Doggy Talents] (DoggyTalents-1.12-1.14.1.240-universal.jar) 
	UC	dsurround{3.4.6.0} [Dynamic Surroundings] (DynamicSurroundings-1.12-3.4.6.0.jar) 
	UC	presets{3.4.6.0} [Presets!] (DynamicSurroundings-1.12-3.4.6.0.jar) 
	UC	enchantingbottles{1.0} [Enchanting Bottles] (enchantingbottles-1.0.jar) 
	UC	csb_ench_table{1.1.2-beta2} [Lapis Stays in the Enchanting Table] (EnchantingTable-1.12-1.1.2-beta2.jar) 
	UC	enchdesc{1.0.4} [Enchantment Descriptions] (EnchantmentDescriptions-1.12-1.0.4.jar) 
	UC	equal_dragons{1.0} [EqualDragons] (EqualDragons-1.0.jar) 
	UC	tschipplib{1.1} [TschippLib] (TschippLib MC1.12 v1.1.jar) 
	UC	extraambiance{1.0.1} [Extra Ambiance] (ExtraAmbiance MC1.12 v1.0.1.jar) 
	UC	fairylights{2.1.1} [Fairy Lights] (fairylights-2.1.1-1.12.jar) 
	UC	friendlyfire{1.5.5} [Friendly Fire] (FriendlyFire-1.12-1.5.5.jar) 
	UC	ftbl{0.0.0} [FTBLib] (FTBLib-4.0.5.jar) 
	UC	ftbu{0.0.0} [FTBUtilities] (FTBUtilities-4.0.3.jar) 
	UC	gravestone{1.8.4} [Gravestone Mod] (gravestone-1.8.4.jar) 
	UC	hopperducts{1.5} [Hopper Ducts] (hopperducts-mc1.12-1.5.jar) 
	UC	incontrol{3.6.0} [InControl] (incontrol-1.12-3.6.0.jar) 
	UC	integrateddynamics{0.8.3} [Integrated Dynamics] (IntegratedDynamics-1.12-0.8.3.jar) 
	UC	integrateddynamicscompat{1.0.0} [IntegratedDynamics-Compat] (IntegratedDynamics-1.12-0.8.3.jar) 
	UC	jei{4.7.5.83} [Just Enough Items] (jei_1.12-4.7.5.83.jar) 
	UC	jepb{1.2} [Just Enough Pattern Banners] (jepb-1.12-1.2.jar) 
	UC	jaff{1.7_for_1.12} [Just a Few Fish] (JustAFewFish-1.7_for_1.12.jar) 
	UC	jeresources{0.8.3.23} [Just Enough Resources] (JustEnoughResources-1.12-0.8.3.23.jar) 
	UC	kleeslabs{5.3.2} [KleeSlabs] (KleeSlabs_1.12-5.3.2.jar) 
	UC	malisiscore{1.12-6.0.2} [MalisisCore] (malisiscore-1.12-6.0.2.jar) 
	UC	malisisblocks{1.12-6.0.0} [Malisis Blocks] (malisisblocks-1.12-6.0.0.jar) 
	UC	manawell{0.0.2} [Mana Wells] (manawell-1.12--0.0.2.jar) 
	UC	mendingfix{1.0.1} [Mending Fix] (mendingfix-1.0.1.jar) 
	UC	netherchest{0.2.1} [Nether Chest] (netherchest-0.2.1.jar) 
	UC	nex{2.1.11.12} [NetherEx] (NetherEx-1.12-2.1.11.12.jar) 
	UC	nonausea{1.0} [NoNausea] (NoNausea-MC1.12-1.0.jar) 
	UC	samsocean{1.0.1} [Ocean Floor Classic] (OceanFloor-1.12-1.0.1.jar) 
	UC	passableleaves{6.1.0.0} [Passable Leaves] (PassableLeaves-1.12-6.1.0.0.jar) 
	UC	progressivedifficulty{1.12_1.3.1} [Progressive Difficulty] (ProgressiveDifficulty-1.12_1.3.1.jar) 
	UC	quickleafdecay{1.2.3} [Quick Leaf Decay] (QuickLeafDecay-MC1.12-1.2.3.jar) 
	UC	reauth{3.5.1} [ReAuth] (ReAuth-3.5.1.jar) 
	UC	roots{0.025} [Roots 2] (roots-2-0.025.jar) 
	UC	ruins{16.7} [Ruins Mod] (Ruins-1.12.jar) 
	UC	rustic{0.4.1} [Rustic] (rustic-0.4.1.jar) 
	UC	saltmod{1.12_d} [Salty Mod] (SaltyMod_1.12_d.jar) 
	UC	scannable{1.4.1.6} [Scannable] (Scannable-MC1.12-1.4.1.6.jar) 
	UC	skyblocks{0.1.0-beta-1.12} [Skyblocks] (Skyblocks-0.1.0-beta-1.12.jar) 
	UC	soundfilters{0.10_for_1.12} [Sound Filters] (SoundFilters-0.10_for_1.12.jar) 
	UC	stepup{4.0-mc1.12} [StepUp] (StepUp-4.0-mc1.12.jar) 
	UC	stg{1.12-1.2.1} [SwingThroughGrass] (stg-1.12-1.2.1.jar) 
	UC	structuredcrafting{0.1.8} [Structured Crafting] (StructuredCrafting-1.12-0.1.8.jar) 
	UE	stupidthings{1.1.2} [Stupid Things] (Stupid Things-1.12-1.1.2.jar) 
	UC	tis3d{1.2.2.4} [TIS-3D] (TIS-3D-MC1.12-1.2.2.4.jar) 
	UC	tnt_utilities{1.2.2} [TNTUtils] (tnt_utilities-mc1.12-1.2.2.jar) 
	UC	toolbelt{1.7.1} [Tool Belt] (ToolBelt-1.12.0-1.7.1.jar) 
	UC	villagenames{1.11.2-1.1a} [Village Names] (VillageNames-1.12-1.1b.jar) 
	UC	vtweaks{1.4.11.3} [V-Tweaks] (VTweaks-1.12-1.4.11.3.jar) 
	UC	wearablebackpacks{3.0.4} [Wearable Backpacks] (WearableBackpacks-1.12-3.0.4.jar) 
	UC	withsprinkles{1.08} [With Sprinkles] (With Sprinkles-1.08.jar) 
	UC	gravelores{1.3} [Gravel Ores] (gravelores-1.7b.jar) 
	Loaded coremods (and transformers): 
PLCore (PassableLeaves-1.12-6.1.0.0.jar)
  passableleaves.core.PLTransformer
LoadingPlugin (Quark-r1.2-104.jar)
  vazkii.quark.base.asm.ClassTransformer
AstralCore (astralsorcery-1.12-1.6.1.jar)
  
TNTUtilities Core (tnt_utilities-mc1.12-1.2.2.jar)
  ljfa.tntutils.asm.ExplosionTransformer
MalisisCorePlugin (malisiscore-1.12-6.0.2.jar)
  net.malisis.core.util.chunkcollision.ChunkCollisionTransformer
  net.malisis.core.util.chunkblock.ChunkBlockTransformer
  net.malisis.core.renderer.transformer.MalisisRendererTransformer
  net.malisis.core.renderer.icon.asm.TextureMapTransformer
  net.malisis.core.util.clientnotif.ClientNotifTransformer
MC64836CoreMod (mc64836-0.1.1.jar)
  nge.sse.mc64836.EntityMinecartTransformer
DynamicSurroundingsCore (DynamicSurroundings-1.12-3.4.6.0.jar)
  org.blockartistry.DynSurround.asm.Transformer
DummyPlugin (ForgeEndertech-1.12-3.0.0.0-build.0002.jar)
  
LoadingPlugin (NoNausea-MC1.12-1.0.jar)
  lumien.nonausea.asm.ClassTransformer
```

**Comment by EnchiladaLlama** [comment]
I have the same problem, too, with only this mod and forge.

**Comment by Furgl** [comment]
Sorry about that, this problem is from trying to load from the scala library. It works fine in a development environment, but not when compiled, which is why I didn't catch it sooner. This is fixed in v1.1.3: 
[Stupid Things-1.12-1.1.3.jar.zip](https://github.com/Furgl/Stupid-Things/files/1219105/Stupid.Things-1.12-1.1.3.jar.zip)
